src/test/java/com/google/sps/EventTest.java

```diff
@@ -0,0 +1,281 @@
+      .setCategory(CATEGORY1)
+      .setTags(TAGS1)
+      .setDuration(MINUTES_10)
+      .setLocation(LOCATION1)
+      .setLinks(LINKS1)
+      .setFields(FIELDS10)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+    
+    Event event2 = Event.newBuilder()
+      .setOwnerID(USER_ID1)
+      .setGCalendarID(G_ID2)
+      .setTitle(TITLE2)
+      .setDescription(DESCRIPTION2)
+      .setCategory(CATEGORY2)
+      .setTags(TAGS2)
+      .setDuration(MINUTES_20)
+      .setLocation(LOCATION2)
+      .setLinks(LINKS2)
+      .setFields(FIELDS11)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+
+    Assert.assertFalse(event1.equals(event2));
+    Assert.assertNotEquals(event1, event2);
+  }
+
+  @Test
+  public void testEvent_equals_withDifferentNumberParameters_returnsNotEqual() {
+    Event event1 = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .setGCalendarID(G_ID1)
+      .setTitle(TITLE1)
+      .build();
+    
+    Event event2 = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .setGCalendarID(G_ID1)
+      .setTitle(TITLE1)
+      .setDescription(DESCRIPTION1)
+      .setCategory(CATEGORY1)
+      .setTags(TAGS1)
+      .setDuration(MINUTES_20)
+      .setLocation(LOCATION1)
+      .setLinks(LINKS1)
+      .setFields(FIELDS11)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+
+    Assert.assertNotEquals(event1, event2);
+  }
+
+  @Test
+  public void testEvent_joinEvent_withInvitedUser_returnsTrue() {
+    Event actual_event = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .setInvitedIDs(Arrays.asList(USER_ID2))
+      .build();
+
+    Assert.assertTrue(actual_event.joinEvent(USER_ID2));
+
+    Event expected_event = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .setJoinedIDs(new ArrayList<String>(Arrays.asList(USER_ID2)))
+      .build();
+
+    Assert.assertEquals(expected_event, actual_event);
+  }
+
+  @Test
+  public void testEvent_joinEvent_withNotInvitedUser_returnsFalse() {
+    Event actual_event = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .build();
+
+    Assert.assertFalse(actual_event.joinEvent(USER_ID2));
+
+    Event expected_event = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .build();
+
+    Assert.assertEquals(expected_event, actual_event);
+  }
+
+  @Test
+  public void testEvent_joinEvent_withOwner_returnsFalse() {
+    Event actual_event = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .build();
+
+    Assert.assertFalse(actual_event.joinEvent(USER_ID1));
+
+    Event expected_event = Event.newBuilder()
+      .setID(EVENT_ID1)
+      .setOwnerID(USER_ID1)
+      .build();
+
+    Assert.assertEquals(expected_event, actual_event);
+  }
+
+  @Test
+  public void testEvent_hasAccessToEvent_withDifferentStatus() {
```

Comment:
Please change the name of the method `userHasAccessToEvent` to something like `hasUserAccessToEvent`.
Maybe just name the test `testEvent_hasUserAccessToEvent` without any further specification.